### DISCUSSION TITLE
Is there a way to suppress the spinner and progress report?

### DISCUSSION THREAD

**Discussion by ammarghaus** [ankitpokhrel/jira-cli#513] [Q&A] [answered]
On every command the the progress is reported with messages like `Fetching ....` with or without a spinner.

Is there a way to suppress this behaviour? Especially when using the `--plain` flag.

The expected behaviour would be that command outputs only the final result in plain text after the necessary delay.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @ammarghaus, you can do something like this

```sh
jira issue list --plain 2>/dev/null
```

**Reply by ammarghaus** [reply]
Thanks a bunch!
